dynamicforms/viewsets.py

```diff
@@ -38,8 +39,10 @@ def new_object(self: viewsets.ModelViewSet):
         # Not that easy: the returned record may not validate for its (correctly) empty fields
         # Maybe we will have to run JavaScript onchange for all fields displayed to ensure at least some consistency?
         # If we do not, subsequent validation may fail because a hidden field has a value
-        field_names = [(f.name + '_id') if isinstance(f, models.ForeignKey) else f.name
-            for f in self.get_queryset().model._meta.fields]
+        field_names = [
+            (f.name + '_id') if isinstance(
```

Comment:
Tukajle bi bilo mnogo bolj jasno gledat kodo, ki bi imela v prvi vrstivi tuple, v drugi vrstici if ... else in v tretji vrstifi for ... in ...

Reply:
popravljeno